pkg/plugins/golang/go_version.go

```diff
@@ -33,10 +33,10 @@ var (
 		major: 1,
 		minor: 13,
 	}
-	go116alpha1 = goVersion{
-		major:      1,
-		minor:      16,
-		prerelease: "alpha1",
+	go116 = goVersion{
+		major: 1,
+		minor: 16,
+		patch: 5,
```

Comment:
Make the variable name generic since we'll have to change this again in the future, and use 1.17 with strict `>` below.
```suggestion
	goVerMax = goVersion{
		major: 1,
		minor: 17,
```

Reply:
Fixed.